src/http_rest_api_handler.cpp

```diff
@@ -652,7 +648,7 @@ Status HttpRestApiHandler::processModelMetadataKFSRequest(const HttpRequestCompo
 static Status parseInferenceHeaderContentLength(HttpRequestComponents& requestComponents,
     const std::vector<std::pair<std::string, std::string>>& headers) {
     for (auto& header : headers) {
-        if (header.first == "Inference-Header-Content-Length") {
+        if (header.first == "inference-header-content-length") {  // drogon automatically converts all headers to lowercase
```

Comment:
Shouldn't we replace it for case insensitive comparison to work with all implementations?

Reply:
yes, we could, but this is 1 more operation on perf critical path

Reply:
did it anyway (to be able to support but net_http and drogon)